runtime/src/bank.rs

```diff
@@ -8119,9 +8128,39 @@ impl Bank {
         }
     }
 
+    fn replace_account<U, V>(
+        &mut self,
+        old_address: &Pubkey,
+        new_address: &Pubkey,
+        old_account: Option<&U>,
+        new_account: &V,
```

Comment:
I found the usage of "old" and "new" here confusing. This function also does more than a replacement, it also deletes an account.

I feel it would be more readable if named like `move_account(src_address, src_account, dst_address, dst_account)`